bin/tailor

```diff
@@ -22,7 +22,12 @@ foreach ([__DIR__ . '/../vendor/autoload.php', __DIR__ . '/../../../autoload.php
 \call_user_func(static function() {
     foreach ([ __DIR__ . '/../.env', __DIR__ . '/../../../../.env', __DIR__ . '/../../../../../.env'] as $file) {
         if (file_exists($file)) {
-            (new Symfony\Component\Dotenv\Dotenv())->usePutenv(true)->loadEnv($file);
+            if (method_exists(Symfony\Component\Dotenv\Dotenv::class, 'usePutenv')) {
```

Comment:
To stay consistent, i would use an absolute namespace here as well `\Symfony\Component\Dotenv\Dotenv`